camel-k-runtime-groovy/src/main/groovy/org/apache/camel/k/groovy/GroovyRoutesLoader.groovy

```diff
@@ -36,6 +36,9 @@ class GroovyRoutesLoader implements RoutesLoader {
         return new RouteBuilder() {
             @Override
             void configure() throws Exception {
+                Builder.delegate = this
```

Comment:
this is problematic if you load multiple groovy files as the static field is going to be overridden

Reply:
Yeah, I expected that the "script.run()" below run till the end, so it's true sequential run. I know this is not the best approach ever ;)